lib/cartopy/examples/lines_and_polygons/README.txt

```diff
@@ -0,0 +1,4 @@
+Lines and polygons
```

Comment:
All of these docstring changes are just to add a title + description, which gets rendered by sphinx-gallery